scripts/Muon/GUI/ElementalAnalysis/elemental_analysis.py

```diff
@@ -364,7 +367,29 @@ def select_data_file(self):
                 self.ptable.select_element(element)
             else:
                 self._remove_element_lines(element)
-        # self._generate_element_data()
+        self._update_checked_data()
+        #self._generate_element_data()
```

Comment:
```suggestion
```

Reply:
There is also a related line above that links to issue #25326 which has been since closed